# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/scheduler/tentative/yield/yield-priority-timers.any.js.ini
# Commit: 134228f242de
# Full Hash: 134228f242dee041e94824d207c4f0e68b48c83c
# Author: Sean Feng <sefeng@mozilla.com>
# Date: 2025-04-01 09:40:15
# Regressor Bug: 1920115
# File Overlap Count: 2
# Description:
#   Bug 1920115 - Allow high priority tasks to run before timers r=smaug
#   
#   In Gecko, unlike other runnables where the TaskController
#   picks one each at a time, the TimeoutManager allows multiple
#   timers to run sequentially without allowing other runnables
# ==============================================================================

diff -r 41d90165c19a -r 134228f242de testing/web-platform/meta/scheduler/tentative/yield/yield-priority-timers.any.js.ini
--- a/testing/web-platform/meta/scheduler/tentative/yield/yield-priority-timers.any.js.ini	Mon Mar 31 19:49:03 2025 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,8 +0,0 @@
-[yield-priority-timers.any.worker.html]
-  [yield() with timer tasks (inherit signal)]
-    expected: FAIL
-
-
-[yield-priority-timers.any.html]
-  [yield() with timer tasks (inherit signal)]
-    expected: FAIL
